src/screens/SendSheet.js

```diff
@@ -702,7 +707,8 @@ export default function SendSheet(props) {
       isEmpty(gasFeeParamsBySpeed) ||
       !selectedGasFee ||
       isEmpty(selectedGasFee?.gasFee) ||
-      !toAddress
+      !toAddress ||
+      l1GasFeeOptimism === null
```

Comment:
wait for optimism gas data to be loaded so we don't prematurely display "insufficient eth"